packages/commons-server/src/libs/response-rules-interpreter.ts

```diff
@@ -198,4 +203,27 @@ export class ResponseRulesInterpreter {
       bodyRaw: this.request.stringBody
     };
   }
+
+  /**
+   * Parse the value using the template parser allowing data helpers.
+   *
+   * @param value the value to parse
+   * @returns the parsed value or the unparsed input value if parsing fails
+   */
+  private parseValue(value: string): string {
+    let parsedValue: string;
+    try {
+      parsedValue = TemplateParser(
+        false,
+        value,
+        this.environment,
+        this.processedDatabuckets,
+        this.request
+      );
+    } catch (error) {
+      return value;
```

Comment:
I thought it makes sense to simply use the unparsed value if parsing fails. Any better idea?

Reply:
No, this makes sense.